its/plugin/src/integrationTest/java/org/sonarsource/iac/MeasuresTest.java

```diff
@@ -34,26 +38,42 @@ class MeasuresTest extends TestBase {
   @CsvSource({
     "cloudformationYamlMeasures, cloudformation, file1.yaml, 2, 5, 1, 1=1;4=1;5=1;6=1;7=1",
     "cloudformationJsonMeasures, cloudformation, file1.json, 2, 9, 0, 1=1;2=1;3=1;4=1;5=1;6=1;7=1;8=1;9=1",
+  })
+  @ResourceLock("measures/cloudformation")
```

Comment:
Key in `ResourceLock` can be arbitrary; tests with the same key will be run seequentially.